Technical/Performance-Benchmarking/performance-benchmarking-of-arangodb-single-instance-vs-split-databases-vs-clustering.md

```diff
@@ -0,0 +1,2030 @@
+# Performance Benchmarking of ArangoDB: Single Instance vs. Split Databases vs. Clustering
+
+<span style="font-size: 34px;">**Table of Contents**</span>
+<span style="font-size: 34px;">**Table of Contents**</span>
+- [Summary](#summary)
+- [Introduction](#introduction)
+  - [Background](#background)
+  - [Objective](#objective)
+  - [Scope](#scope)
+- [Cost Analysis](#cost-analysis)
+  - [Objective](#objective-1)
+  - [Scope](#scope-1)
+  - [Cost Components Considered](#cost-components-considered)
+  - [Cost Breakdown for Each Configuration](#cost-breakdown-for-each-configuration)
+    - [Single Instance](#single-instance)
+    - [2 Split Databases](#2-split-databases)
+    - [4 Split Databases](#4-split-databases)
+    - [Clustering](#clustering)
+- [Methodology](#methodology)
+  - [Configuration Details](#configuration-details)
+  - [Benchmarking Criteria](#benchmarking-criteria)
+- [Results and Analysis](#results-and-analysis)
+  - [Performance Metrics](#performance-metrics)
+    - [Single Instance](#single-instance-1)
+    - [2 Split Databases](#2-split-databases-1)
+    - [4 Split Databases](#4-split-databases-1)
+    - [Clustering](#clustering-1)
+  - [Analysis](#analysis)
+  - [JSON End-to-End Times](#json-end-to-end-times)
+- [Next Steps](#next-steps)
+- [Screenshots](#screenshots)
+  - [Single Instance](#single-instance-2)
+  - [2 Split Databases](#2-split-databases-2)
+  - [4 Split Databases](#4-split-databases-2)
+  - [Clustering](#clustering-2)
+
+
+<span style="font-size: 34px;">**Summary**</span>
+
+This report presents a comprehensive performance benchmarking analysis of ArangoDB deployed in three distinct configurations on Azure: <span style="color:red;">**a Single Instance, Split Databases, and Clustering.**</span>  The aim was to include the best performing configurations and the most cost-efficient setups. It will highlight the FTPS value as a critical performance metric and its impact on choosing the right configuration for deployment.
+
+The highest **FTPS** that was reached was <span style="color:green; font-size: 16px;">****2496.83****</span> using the **4 SplitDB** approach of Arango.
```

Comment:
The costs are in this document already which were can refer to in the future.

There were other changes made since the test with the first splitdbs , but they were present with the clustered version benchmarking